frontend/components/login/register.tsx

```diff
@@ -27,10 +28,20 @@ export function RegisterHeader({ goToLogin }: { goToLogin: () => void }) {
       <Flex direction="column" align="center">
          <Heading mt="8px">Create Account</Heading>
          <Flex mt="12px" align="baseline">
-            <Text m="0 4px 0 0" fontWeight="normal" fontSize="14px" color="var(--color-gray-6)">
+            <Text
+               m={`0 ${space[1]} 0 0`}
+               fontWeight="normal"
+               fontSize={`${fontSize.md}`}
+               color={`${color.gray6}`}
+            >
                Been here before?
             </Text>
-            <Button variant="link" fontSize="14px" color="var(--color-blue)" onClick={goToLogin}>
+            <Button
+               variant="link"
+               fontSize={`${fontSize.md}`}
+               color={`${color.blue}`}
```

Comment:
In `options.tsx` you had replaced `--color-blue` with `backgroundColor={'${color.blue['ifixit']}'}` 

Are these just different shades of the same blue?

Reply:
Yes, they are slightly different! Here's the JSON for the core-primitives, for future reference: https://github.com/iFixit/core-primitives/blob/master/index.json

Reply:
Oh thanks!